pkgs/applications/misc/huggle/default.nix

```
@@ -0,0 +1,141 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, ncurses
+, which
+, cmake
+, unzip
+, qt5
+, libyamlcpp
+}:
+
+let
+  libirc = fetchFromGitHub {
+    owner = "grumpy-irc";
+    repo = "libirc";
+    rev = "65b0277a9f6bf8dc9c3412f17d6b2ebab90344d0";
+    sha256 = "u6hG3vycrvSdEDQ4R7ja/RTXfm1IkEenDEtAhZLei+8=";
+  };
+  enwiki = fetchFromGitHub {
+    owner = "huggle";
+    repo = "enwiki";
+    rev = "b772daf89b9c7d765aadc3206e9b121d116469b7";
+    sha256 = "ceS1suwBKhTytYNmCZvD7U+HIrUp71LPZydOyMqFtAE=";
+  };
+  extension-flow = fetchFromGitHub {
+    owner = "huggle";
+    repo = "extension-flow";
+    rev = "e5e4beaebdf5b5b7d17043fedc8201078f469ada";
+    sha256 = "MnQFd4QLnW+2Iiv/wsJV/T9jvViusX+EYXsUCNxdLi4=";
+  };
+  extension-mass-delete = fetchFromGitHub {
+    owner = "huggle";
+    repo = "extension-mass-delete";
+    rev = "f421307b35bb58b49c5f7b38245936505fd516d9";
+    sha256 = "xxnBTjx1cwwUS1QTNsdveMefM1j0a6jjtOSuBja6N9o=";
+  };
+  extension-scoring = fetchFromGitHub {
+    owner = "huggle";
+    repo = "extension-scoring";
+    rev = "3378982ad67d7f7a52c77d1dec5a56eee9c12acd";
+    sha256 = "pxVGAvXChPdCi2wVq8aFqQrliQAoKySbWVYriKCB0no=";
+  };
+  extension-splitter-helper = fetchFromGitHub {
+    owner = "huggle";
+    repo = "extension-splitter-helper";
+    rev = "557728a13b87d6d9d36a08411482a153ee1a726d";
+    sha256 = "UkxlGCY55lE799GnOy7zjWYjqqfNCpf2o+Rqut2ffak=";
+  };
+  extension-thanks = fetchFromGitHub {
+    owner = "huggle";
+    repo = "extension-thanks";
+    rev = "ac700920553ae8dc188eb9396cc4a23d070ba661";
+    sha256 = "EMjwSXdWc83ijlZj0HUFXpuJO3GJZTTHX/8ubLKCf+8=";
+  };
+  mass-delivery = fetchFromGitHub {
+    owner = "huggle";
+    repo = "mass-delivery";
+    rev = "7b936b3e557a152064e5b05b1418a132271485ac";
+    sha256 = "G1QNJpbNTNY6NQqNxLn68glnQEWKKZ5prINhUGPSOO0=";
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "huggle";
+  version = "3.4.10";
+  src = fetchFromGitHub {
+    owner = "huggle";
+    repo = "huggle3-qt-lx";
+    rev = version;
+    sha256 = "jgheTcUsBeFIqgO2sGmBsjlaxG8LseScWAxfdgW2AJA=";
+  };
+
+  nativeBuildInputs = [
+    qt5.qtbase
+    qt5.wrapQtAppsHook
+    qt5.qtmultimedia
+    qt5.qtwebkit
+    qt5.qtwebengine
+    pkg-config
+    ncurses
+    which
+    cmake
+    libyamlcpp
```

```suggestion
    wrapQtAppsHook
    pkg-config
    which
    cmake
```
Those go all to buildInputs. Also with libsForQt5 we can replace qt5 with the wrapQtAppsHook etc.